deployment/kubernetes/hub/3-alertdb.yml

```diff
@@ -0,0 +1,68 @@
+kind: List
+apiVersion: v1
+items:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: alertdb
+    spec:
+      selector:
+        matchLabels:
+          app: alertdb
+          tier: alertdb
+      replicas: 1
+      template:
+        metadata:
+          name: alertdb
+          labels:
+            app: alertdb
+            tier: alertdb
+        spec:
+          volumes:
+            # NOTE: replace the emptyDir volume with a persistent volume or all the data will be lost when the pod is destroyed.
+            - emptyDir: {}
```

Comment:
Should we document and/or parameterize these since they are shared?

Reply:
I think once the helm charts are finalized we should document that approach.  I will add to the comment here.